src/compose_notifications/main.py

```diff
@@ -1980,6 +2000,12 @@ def get_triggering_function(message_from_pubsub):
 
     return triggered_by_func_id
 
+def delete_ended_search_following(conn, new_record) #issue425
+    """Delete from user_pref_search_whitelist if the search goes to one of ending statuses"""
```

Comment:
my syntax highlight suggests that there's some mismatch with """ usage, which causes the following codes to be inverted: comments are treated as code, code treated as comments. could you pls double-check.

Reply:
Retyped """, don't see a problem

Reply:
corrected